sechub-pds/src/test/java/com/mercedesbenz/sechub/pds/autocleanup/PDSAutoCleanupServiceTest.java

```diff
@@ -31,12 +31,12 @@ void beforeEach() {
 
         configService = mock(PDSConfigService.class);
         jobRepository = mock(PDSJobRepository.class);
-        timeCalculationService = mock(TimeCalculationService.class);
+        PDSTimeCalculationService = mock(PDSTimeCalculationService.class);
         inspector = mock(PDSAutoCleanupResultInspector.class);
 
         serviceToTest.configService = configService;
         serviceToTest.jobRepository = jobRepository;
-        serviceToTest.timeCalculationService = timeCalculationService;
+        serviceToTest.PDSTimeCalculationService = PDSTimeCalculationService;
```

Comment:
fields have to start with a lower case character - please rename it inside the service  class and also inside this test - I think the IDE refactoring did the fault here...
```suggestion
        serviceToTest.pDSTimeCalculationService = PDSTimeCalculationService;
```